src/streaming/controllers/AbrController.js

```diff
@@ -368,6 +372,10 @@ function AbrController() {
             idx = _checkMaxBitrate(type, streamId);
             idx = _checkMaxRepresentationRatio(idx, type, streamId);
             idx = _checkPortalSize(idx, type, streamId);
+            // Apply maximum suggested bitrate from CMSD headers if enabled 
+            if (settings.get().streaming.cmsd.abr.applyMb) {
```

Comment:
Do we need to check for `cmsd.enabled`here?

Reply:
We don't necessarily have to check since if cmsd.enabled is false, then mb would never be set.
However, for consistency and to avoid requesting CmsdModel we can add the check.